src/ace/managers/system.c

```diff
@@ -593,6 +649,15 @@ void systemUnuse(void) {
 
 	--s_wSystemUses;
 	if(!s_wSystemUses) {
+		// Disable CDTV CD drive if present
```

Comment:
Perhaps this should be after systemFlushIo() 'cuz I think it works for all floppy/hdd/cd I/O, but I'm not quite sure.

Reply:
done